content/en/tracing/profiler/profiler_troubleshooting.md

```diff
@@ -209,12 +209,22 @@ variable to `1`, as described in the
 
 Without this flag, profiles for short-lived Resque jobs will be unavailable.
 
+## Profiling does not turn on because compilation of the Ruby VM just-in-time header failed
+
+There is a known incompatibility between Ruby 2.7 and older GCC versions (4.8 and below) that impacts the profiler ([upstream Ruby report][6], [`dd-trace-rb` bug report][7]). This can result in the following error message: "Your ddtrace installation is missing support for the Continuous Profiler because compilation of the Ruby VM just-in-time header failed. Your C compiler or Ruby VM just-in-time compiler seem to be broken."
+
+
+To fix this, update your operating system, Docker image, or GCC version to something more recent than v4.8.
```

Comment:
On a second look at this, is it just me or does this sentence read like you can update the operating system or docker image to "v4.8" whereas the "v4.8" only refers to the GCC version?

Reply:
```suggestion
To fix this, update your operating system or Docker image so that the GCC version is something more recent than v4.8.
```